backbone.js

```diff
@@ -87,6 +87,9 @@
         return _[method](this[attribute], cb(iteratee, this), context);
       };
       case 4: return function(iteratee, defaultVal, context) {
+        if(arguments.length < 2) {
+          return _[method](this[attribute], cb(iteratee, this));
+        }
```

Comment:
Rather than add another condition here, I'd rather deopt `reduce` and friends into the `default` case below. Mind updating the [`collectionMethods`](https://github.com/jashkenas/backbone/blob/master/backbone.js#L1169-L1176) object, giving `reduce`, `inject`, `foldl`, `reduceRight` and `foldr` a value of `0`?